packages/core/src/AvatarGroup/AvatarGroupContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { createContext, useContext, useMemo } from "react";
+
+type HvAvatarGroupContextProviderProps = {
+  size: "xs" | "sm" | "md" | "lg" | "xl";
+  children: React.ReactNode;
+};
+
+type HvAvatarGroupContextProp = {
+  size: "xs" | "sm" | "md" | "lg" | "xl";
+};
+
+export const HvAvatarGroupContext =
+  createContext<HvAvatarGroupContextProp | null>(null);
+
+const HvAvatarGroupProvider = ({
+  size,
+  children,
+}: HvAvatarGroupContextProviderProps) => {
+  const value = useMemo(() => ({ size }), [size]);
+
+  return (
+    <HvAvatarGroupContext.Provider value={value}>
+      {children}
+    </HvAvatarGroupContext.Provider>
+  );
+};
+
+export const useAvatarGroupContext = () => {
+  const context = useContext(HvAvatarGroupContext);
+  return context;
+};
+
+export default HvAvatarGroupProvider;
```

Comment:
default exports are bad [[1]](https://basarat.gitbook.io/typescript/main-1/defaultisbad) [[2]](https://medium.com/@islizeqiang/why-you-should-avoid-default-exports-in-javascript-modules-7d087dde7ac6)